### DISCUSSION TITLE
Am i going crazy or can you not miss a note by spamming now?

### DISCUSSION THREAD

**Discussion by macohi** [FunkinCrew/Funkin#3878] [Q&A] [answered]
Before VSlice when you hit a note when there are no notes you would lose your combo.

But right now while playing I keep hitting notes when there aren't any there (accidentally) and I don't appear to be losing my combo, I thought this was a bug and is a one time thing but it continues, is this intentional or no?

ANSWERED by @KoloInDaCrib!
[answer here](https://github.com/FunkinCrew/Funkin/discussions/3878#discussioncomment-11311404)

**Comment by KoloInDaCrib** [comment, marked as answer]
It's intentional, function `ghostNoteMiss` (called upon pressing a key without any available notes) doesn't call `applyScore` (responsible for score counting), unlike the function `onNoteMiss` (self explanatory function) 

This was a good decision in terms of game design, as you would likely not want players being frustrated that they didnt get a better rank due to doing ghost misses

**Reply by macohi** [reply]
Good to know! Thanks!

**Reply by macohi** [reply]
> It's intentional, function `ghostNoteMiss` (called upon pressing a key without any available notes) doesn't call `applyScore` (responsible for score counting), unlike the function `onNoteMiss` (self explanatory function)

wait doesnt this count as ghost tapping?
just with the miss animation?

**Reply by KoloInDaCrib** [reply]
> wait doesnt this count as ghost tapping?
just with the miss animation?

![Screenshot_20241119-235238~2](https://github.com/user-attachments/assets/5cc8e0f8-bcf2-442b-b30d-6ed0286ba70a)

Yeah

**Reply by macohi** [reply]
LOL
